Model/Behavior/UploadBehavior.php

```diff
@@ -1344,7 +1344,7 @@ protected function _resizePhp(Model $model, $field, $path, $size, $geometry, $th
 				$outputHandler = 'imagepng';
 				$supportsQuality = true;
 				// convert 0 (lowest) - 100 (highest) thumbnailQuality, to 0 (highest) - 9 (lowest) quality (see http://php.net/manual/en/function.imagepng.php)
-				$adjustedQuality = intval((100 - $this->settings[$model->alias][$field]['thumbnailQuality']) / 100 * 9);
+				$adjustedQuality = (int)((100 - $this->settings[$model->alias][$field]['thumbnailQuality']) / 100 * 9);
```

Comment:
Hi,
If it works well intval() in PHP 5, but is not recommended to use more it is a function deprecator. Also in the same code is being used cast (int) and for better code quality must remain consistent.

Example: 
UploadBehavior.php 
Lines 508, 605.630, etc ...